style.css

```diff
@@ -0,0 +1,145 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* source: https://gist.github.com/BrendonKoz/b1df234fe3ee388b402cd8e98f7eedbd */
+/* author: BrendonKoz */
+:root {
+	--map-tiles-filter: brightness(0.6) invert(1) contrast(3) hue-rotate(200deg)
+		saturate(0.3) brightness(0.7);
+}
+
+@media (prefers-color-scheme: dark) {
+	.map-tiles {
+		filter: var(--map-tiles-filter, none);
+	}
+}
+
+.error {
+	display: flex;
+	justify-content: center;
+	width: 100vw;
+}
+
+.box {
```

Comment:
You can use classList.add() or className to add tailwind to your dom-manip file or with your back-tics just like the normal HTML tailwind usage.